packages/components/psammead-useful-links/src/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components';
+import { getSerifMedium } from '@bbc/psammead-styles/font-styles';
+import { C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { grid } from '@bbc/psammead-styles/detection';
+import { getPica } from '@bbc/gel-foundations/typography';
+import { string, shape, node } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import {
+  GEL_SPACING, // 8 px
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+
+export const UsefulLinkItem = styled.a`
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => getSerifMedium(service)};
```

Comment:
```suggestion
  ${({ service }) => service && getSerifMedium(service)};
```
To fail safely if `service` is not provided. Even though it is a required prop.